src/shared/prerendered-app/Intro/style.css

```diff
@@ -247,3 +248,13 @@
     display: none;
   }
 }
+
+.sample-button {
+  &:focus {
+    outline: 0;
+  }
+
+  &:focus .demo-icon-container {
+    box-shadow: 0 0 0 4px var(--deep-blue), 0 0 0 8px var(--pink);
+  }
```

Comment:
Feels like we could use `focus-visible` for these?

Reply:
I’m not sure how would that help. What do you have in mind? :)

Reply:
It would avoid showing the focus outline during mouse interaction.

Reply:
Fair enough. :) Applied!